Orange/widgets/utils/localization.py

```diff
@@ -0,0 +1,15 @@
+__all__ = ["pl", "plsi"]
+
+
+def pl(n, forms):
```

Comment:
Hm, is it really "101 instance"?

Reply:
Oops. I don't know where I got this from. :) Fixed.